server/src/main.rs

```diff
@@ -31,7 +31,7 @@ async fn main() -> anyhow::Result<()> {
 
     let cors = CorsLayer::new()
         .allow_methods([Method::GET, Method::POST])
-        .allow_origin(Any);
+        .allow_origin("https://mina.vote/".parse::<HeaderValue>().unwrap());
```

Comment:
This doesn't take into consideration the staging URL. I think development env should allow `Any`

Reply:
Good point. I'm working on adding this

Reply:
https://docs.rs/tower-http/latest/tower_http/cors/struct.CorsLayer.html#method.allow_origin